scripts/main.js

```diff
@@ -109,7 +108,7 @@ function update() {
 
 		let tileFood = map[pos[0]][pos[1]].food / maxTileFood;
 		let age = (creature.age / (1000 / agingSpeed));
-		let reproduceTime = creature.reproduceTime / (minReproduceTime * 2.5);
+		let reproduceTime = creature.reproduceTime / (minReproduceTime * 2.5);	// These 3 variables are assigned but never used?
```

Comment:
Some of these could be useful but I'm not quite sure yet that's why they're defined but not used.